src/csharp/Microsoft.Spark.Worker/Command/RDDCommandExecutor.cs

```diff
@@ -112,7 +112,9 @@ internal class RDDCommandExecutor
             {
                 case CommandSerDe.SerializedMode.Byte:
                     BinaryFormatter formatter = s_binaryFormatter ??= new BinaryFormatter();
+#pragma warning disable SYSLIB0011 // Type or member is obsolete
```

Comment:
I worry about the continued use of BinaryFormatter due to https://learn.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-security-guide.  pragma'ing away is necessary to compile...but I worry this needs to be fixed soon.

Reply:
Yes we will need to fix this in upcoming PRs